README.md

```diff
@@ -60,6 +60,13 @@ to generate the private key to interactively log in to EC2 instances.
 Alternatively you may select **None** for the SSH credentials and the plugin
 will dynamically manage the SSH key pair used by each instance.
 
+```
+If you decide to switch from dynamic key management to using a static SSH
+credential it is advisable that you terminate and restart any running agent
+instances, otherwise the controller will not be able to re-attach to them
+if they are disconnected.
+```
```

Comment:
Why was this a code block?

```suggestion
If you decide to switch from dynamic key management to using a static SSH
credential it is advisable that you terminate and restart any running agent
instances, otherwise the controller will not be able to re-attach to them
if they are disconnected.
```